tests/index.test.js

```diff
@@ -0,0 +1,25 @@
+const describe = require('mocha').describe
```

Comment:
Really solid tests here, you hit the four scenarios. It was expected that these would be committed first though.